tf2_geometry_msgs/include/tf2_geometry_msgs/tf2_geometry_msgs.h

```diff
@@ -186,73 +183,6 @@ void fromMsg(const geometry_msgs::PointStamped& msg, geometry_msgs::PointStamped
 }
 
 
-/*****************/
-/** PoseStamped **/
-/*****************/
-
-/** \brief Extract a timestamp from the header of a Pose message.
- * This function is a specialization of the getTimestamp template defined in tf2/convert.h.
- * \param t PoseStamped message to extract the timestamp from.
- * \return The timestamp of the message.
- */
-template <>
-inline
-  const ros::Time& getTimestamp(const geometry_msgs::PoseStamped& t)  {return t.header.stamp;}
```

Comment:
The only change affecting the api is the removing gmTransformToKDL(); every other functions has just been moved in order to avoid some code duplication (git did not get this actually...). Of cause  gmTransformToKDL() can be re-added in order to do the deprecation cycle, if desired.